formats/utils/color-factory/configs/base.js

```diff
@@ -0,0 +1,737 @@
+const {saturationAdjustmentFn, hueRotationFn} = require('../utils');
+
+const base = {
+  surface: [
```

Comment:
I’d like to leave the scheme as-is for now, get a working beta released, then iterate on adding metadata which will be part of the generated files. Specifically the immediate plan for metadata is to add info that can be used to filter colors based on various dimensions.